packages/feedback-service/src/feedback-config/schema.ts

```diff
@@ -4,9 +4,15 @@ import {
 
 export const FeedbackConfigSchema: Schema = new Schema({
   appId: {
+    type: String,
+    required: false,
+    unique: false,
```

Comment:
App Id should be unique

Reply:
I thought we were going to remove it. I'll make it unique again.

Reply:
Done